source/docs/software/frc-glossary.rst

```diff
@@ -68,5 +71,8 @@ FRC Glossary
    teleop
       The second phase of each match is called the Teleoperated Period (teleop) and consists of drivers controlling their robots.
 
+   torque
```

Comment:
Are we actually linking to this anywhere in the docs?

Reply:
Yes, we are now